server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -105,303 +105,305 @@ class AddressIndexRepository @Inject()(
     */
   def generateQueryAddressRequest(tokens: Map[String, String], originalInput: String): SearchDefinition = {
 
+    val defaultFuzziness = "1"
+
     val saoQuery = Seq(
       tokens.get(Tokens.saoStartNumber).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoStartNumber",
           value = token
-        ).boost(queryParams.subBuildingName.lpiSaoStartNumberBoost)),
+        )).boost(queryParams.subBuildingName.lpiSaoStartNumberBoost)),
       tokens.get(Tokens.saoStartSuffix).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoStartSuffix",
           value = token
-        ).boost(queryParams.subBuildingName.lpiSaoStartSuffixBoost)),
+        )).boost(queryParams.subBuildingName.lpiSaoStartSuffixBoost)),
       tokens.get(Tokens.saoEndNumber).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoEndNumber",
           value = token
-        ).boost(queryParams.subBuildingName.lpiSaoEndNumberBoost)),
+        )).boost(queryParams.subBuildingName.lpiSaoEndNumberBoost)),
       tokens.get(Tokens.saoEndSuffix).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoEndSuffix",
           value = token
-        ).boost(queryParams.subBuildingName.lpiSaoEndSuffixBoost))
+        )).boost(queryParams.subBuildingName.lpiSaoEndSuffixBoost))
     ).flatten
 
     val subBuildingNameQuery = Seq(
       tokens.get(Tokens.subBuildingName).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "paf.subBuildingName",
           value = token
-        ).boost(queryParams.subBuildingName.pafSubBuildingNameBoost)),
+        )).boost(queryParams.subBuildingName.pafSubBuildingNameBoost)),
       tokens.get(Tokens.subBuildingName).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoText",
           value = token
-        ).boost(queryParams.subBuildingName.lpiSaoTextBoost)),
+        )).boost(queryParams.subBuildingName.lpiSaoTextBoost)),
       tokens.get(Tokens.saoStartSuffix).map(token =>
-        matchQuery(
+        constantScoreQuery(matchQuery(
           field = "lpi.saoStartSuffix",
```

Comment:
No, I reckon it should. I think we are find for now, but in the long term we need to think how we may wish to group subBuildingName and BuildingName together, if they are joined using "must".